src/status_im2/contexts/shell/jump_to/components/jump_to_screen/view.cljs

```diff
@@ -19,25 +20,16 @@
 
 (defn placeholder
   []
-  [linear-gradient/linear-gradient
-   {:colors [colors/neutral-100-opa-0 colors/neutral-100-opa-100]
-    :start  {:x 0 :y 0}
-    :end    {:x 0 :y 1}
-    :style  (style/placeholder-container (safe-area/get-top))}
-   [rn/image
-    {:source nil ;; TODO(parvesh) - add placeholder image
-     :style  style/placeholder-image}]
-   [quo/text
-    {:size   :paragraph-1
-     :weight :semi-bold
-     :style  style/placeholder-title}
-    (i18n/label :t/shell-placeholder-title)]
-   [quo/text
-    {:size   :paragraph-2
-     :weight :regular
-     :align  :center
-     :style  style/placeholder-subtitle}
-    (i18n/label :t/shell-placeholder-subtitle)]])
+  [rn/view {:style (style/placeholder-container (safe-area/get-top))}
+   [linear-gradient/linear-gradient
+    {:colors [colors/neutral-100-opa-0 colors/neutral-100-opa-100]
+     :start  {:x 0 :y 0}
+     :end    {:x 0 :y 1}
+     :style  (style/placeholder-container (safe-area/get-top))}]
```

Comment:
Maybe we should now rename those styles from `placeholder-...` to something more descriptive since now the placeholder has been removed

Reply:
Hi @ulisesmac, Thank you very much for reviewing PR. We didn't removed the placeholder screen, we just added missing illustration to that screen.